src/applications/vaos/tests/appointment-list/components/RequestedAppointmentDetailsPage.unit.spec.js

```diff
@@ -1,831 +0,0 @@
-    });
-    const screen = renderWithStoreAndRouter(<AppointmentList />, {
-      initialState,
-      path: '/pending',
-    });
-
-    const detailLinks = await screen.findByRole('link', { name: /Details/i });
-
-    fireEvent.click(detailLinks);
-
-    expect(await screen.findByText('Pending primary care appointment')).to.be
-      .ok;
-    expect(
-      screen.getByText('Back to pending appointments').getAttribute('href'),
-    ).to.equal('/pending');
-  });
-
-  it('should show error message when single fetch errors', async () => {
-    // Arrange
-    const response = new MockAppointmentResponse();
-
-    mockAppointmentApi({ response, responseCode: 500 });
-
-    // Act
-    const screen = renderWithStoreAndRouter(<AppointmentList />, {
-      initialState,
-      path: `/pending/${response.id}`,
-    });
-
-    // Assert
-    await waitFor(() => {
-      expect(document.activeElement).to.have.tagName('h1');
-    });
-
-    expect(
-      screen.getByRole('heading', {
-        level: 1,
-        name: 'We’re sorry. We’ve run into a problem',
-      }),
-    ).to.be.ok;
-  });
-
-  it('should display pending document title', async () => {
-    // Arrange
-    const response = new MockAppointmentResponse({
-      status: APPOINTMENT_STATUS.proposed,
-    });
-
-    mockAppointmentApi({ response });
-
-    // Act
-    renderWithStoreAndRouter(<AppointmentList />, {
-      initialState,
-      path: `/pending/${response.id}`,
-    });
-
-    // Assert
-    await waitFor(() => {
-      expect(global.document.title).to.equal(
-        'Request for primary care appointment | Veterans Affairs',
-      );
-    });
-  });
-
-  it('should dispay CC document title', async () => {
-    // Arrange
-    const response = new MockAppointmentResponse({
-      kind: 'cc',
-      serviceType: 'audiology-hearing aid support',
-      status: APPOINTMENT_STATUS.proposed,
-    });
-
-    mockAppointmentApi({ response });
-
-    // Act
-    renderWithStoreAndRouter(<AppointmentList />, {
-      initialState,
-      path: `/pending/${response.id}`,
-    });
-
-    // Assert
-    await waitFor(() => {
-      expect(global.document.title).to.equal(
-        `Request for hearing aid support community care appointment | Veterans Affairs`,
-      );
-    });
-  });
-
-  it('should display cancel document title', async () => {
-    // Arrange
-    const response = new MockAppointmentResponse({
-      status: APPOINTMENT_STATUS.cancelled,
-    });
-    response.setRequestedPeriods([moment()]);
-
-    mockAppointmentApi({ response });
-
-    // Act
-    renderWithStoreAndRouter(<AppointmentList />, {
-      initialState,
-      path: `/pending/${response.id}`,
-    });
-
-    // Assert
-    await waitFor(() => {
-      expect(global.document.title).to.equal(
-        'Canceled request for primary care appointment | Veterans Affairs',
-      );
-    });
-  });
-
-  describe('When "vaOnlineSchedulingAppointmentDetailsRedesign" feature is on', () => {
```

Comment:
Hmm do we need to salvage any of these tests?

Reply:
Yes. Good catch!